target_postgres/tests/test_standard_target.py

```diff
@@ -116,6 +117,42 @@ def remove_metadata_columns(row: dict) -> dict:
     return new_row
 
 
+def verify_data(
```

Comment:
Love the simplicity of this. As a part of this PR (I know this expands the scope a bit I'd be happy if you want to split this to a seperate PR) can we also be sure we're pulling in the SDK tests that have been added. Honestly another file probably makes the most sense (maybe sdk_tests.py or something) See https://github.com/MeltanoLabs/target-snowflake/blob/main/tests/test_target_snowflake.py#L71

Reply:
Created #223